Packs/GoogleSafeBrowsing/Integrations/GoogleSafeBrowsingV2/GoogleSafeBrowsingV2.py

```diff
@@ -0,0 +1,209 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+
+import requests
+import traceback
+from typing import Dict, Any
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+TYPES = {
+    'threatTypes': ["MALWARE", "SOCIAL_ENGINEERING", "POTENTIALLY_HARMFUL_APPLICATION", "UNWANTED_SOFTWARE"],
+    'platformTypes': ["ANY_PLATFORM", "WINDOWS", "LINUX", "ALL_PLATFORMS", "OSX", "CHROME", "IOS", "ANDROID"]
+}
+
+
+class Client(BaseClient):
+    def url_request(self, body: Dict, full_url):
+        result = self._http_request(
+            method='POST',
+            json_data=body,
+            full_url=full_url)
+        return result
```

Comment:
can't you call directly `_http_request`?

Reply:
anyway, building the req dict should be in the client, not in a separate func.

Reply:
done